src/components/TodoList.js

```diff
@@ -0,0 +1,143 @@
+import React, { Component } from "react";
+import "./styles/style.css";
+export default class TodoList extends Component {
+
+  render() {
```

Comment:
This entire file is a mess... clean up your JSX. For one, it's too large ... I'm sure some of that can be split into separate components.